EIPS/eip-4747.md

```diff
@@ -0,0 +1,313 @@
+---
+eip: 4747
+title: Simplify EIP-161
+description: Simplify EIP-161 and retroactively deprecate unused aspects of it
+author: Peter Davies (@petertdavies)
+discussions-to: https://ethereum-magicians.org/t/eip-4747-simplify-eip-161/8246
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-02-02
+requires: 161
+---
+
+
+## Abstract
+
+Simplify the definition of EIP-161, removing the requirement for implementors to support edge cases that are impossible on Mainnet.
+
+## Motivation
+
+EIP-161 is overly complex and has a number of edge cases that are poorly documented and tested. This EIP takes advantage of the complete removal of all remaining empty accounts in block 14,049,881 (tx `0xf955834bfa097458a9cf6b719705a443d32e7f43f20b9b0294098c205b4bcc3d`) to clarify it, and allows implementors to not implement various edge cases that never occurred and are not possible in the future.
```

Comment:
This EIP uses different styles for block numbers: it is comma delimited here and in the rationale, while without delimiters elsewhere.

My impression is that most EIPs do not include commas in block numbers, but it is up to you.